tasks/run-zap/task.sh

```diff
@@ -25,5 +26,6 @@ done
 zap-cli shutdown
 
 echo Uploading files...
-python scripts/tools/s3upload.py
-
+# credentials are provided via environment variables
+# http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html#cli-environment
+aws s3 sync results "s3://$S3_BUCKET/results"
```

Comment:
Oh. Good call! Does this play well with versioned files in the buckets? And, can we do the reverse to sync the most recent version of the files locally? We'll need to do that for the summary file work @ctro is tackling.